docs/modules/device-lab-management/pages/deviceConnect/remote-update-deviceconnect.adoc

```diff
@@ -0,0 +1,81 @@
+= Perform a remote deviceConnect update
+:navtitle: Perform a remote deviceConnect update
+
+Cloud and Hybrid customers hosting dedicated devices can perform a remote update for deviceConnect on the Mac mini host. This action can be performed either manually or automatically from the Kobiton Portal via the Device Management page.
+
+
+== Prerequisites
+
+* Only available for Cloud and Hybrid customers.
+* Gather the internal IP address or hostname of the Mac mini host(s).
+* Kobiton administrator with the ADMIN predefined permission.
+* Remote update is available in deviceConnect version 4.13.0 and above.
+
+== Perform a remote deviceConnect update manually
+
+When the below notification banner appears in the Portal, a deviceConnect update is available for at least 1 Mac mini host in your organisation:
+
+image:new-version-available.png[width=1000,alt="The banner about new version of deviceConnect"]
+
+To update deviceConnect remotely, log into the Kobiton Portal with an administrator account.
+
+include::profile:partial$open-settings.adoc[]
+
+Select the *Device Management* tab.
+
+The *Device Management* page lists all dedicated mobile devices for the org grouped by Mac mini host. Locate the Mac mini host you want to update and select the corresponding *Upgrade* button.
+
+[TIP]
+A Mac mini host that does not have the latest deviceConnect version has a warning icon next to its name in Device Management.
+
+[IMPORTANT]
+Make sure the current deviceConnect version on the Mac mini host is at least 4.13.0 before continuing, as deviceConnect below this version cannot be remotely updated.
```

Comment:
from "cannot be remotely updated" to "cannot be updated via the Portal."

Reply:
updated